docs/conops.md

```diff
@@ -0,0 +1,146 @@
+
+We believe this business model creates a win-win situation where it releases the production and legal burdens from Arrow, and it serves as a direct-to-consumer channel to OEMs, allowing them to monetize on their vehicles with a shorter time-to-market than if they have to sell them through contracts.
+
+
+#### Downside
+While the marketplace business model bridges many gaps on the market amount different participants, the platform (Arrow) itself may under-perform as operating a marketplace is difficult and the success is largely a function [network effects](https://online.hbs.edu/blog/post/what-are-network-effects). 
+
+There essentially three parties in a marketplace ecosystem: the platform (Arrow), the service providers (OEMs or other fleet owner-operators), the customers (passengers, corporations, etc.).
+
+At the early stage, Arrow would have to make outreach efforts to both service providers and customers because a lack of either parties will fail the business. If Arrow were to pursue this business model, a well-thought-out go-to-market strategy is needed and the execution has to be near flawless.
+
+#### Further Discussions
+Key points to discuss:
+* Whether Arrow wants to pursue a marketplace model
+* If we go with the vertical integration model, what kind of operations are we capable of running commercially
+* Who and when to showcase our demo
+* Should we seek strategic investors
+
+
+## Risks and Potential Issues
+***The underlying pricing model may be incorrectly constructed, thereby over- or under-price trips.***
+
+The pricing logic of `svc-pricing` derives from our [pricing model](https://docs.google.com/spreadsheets/d/1mjPtaIn3E5m7r4nyKt_sJKG9BSFm2ty7Gzo7OqERxwo), which was constructed without the assistance and review from external consultants. And currently, the pricing model makes a number of rough estimates on some key inputs like the cost basis of a cargo aircraft. With inaccurate pricing Arrow's business, reputation, results of operations and financial condition can be materially and adversely affected. 
+
+***The pricing service may experience outage.***
+
+Our cloud service providers and other services `svc-pricing` depends on may be impacted by political events, trade and other international disputes, war, terrorism, natural disasters, public health issues, industrial accidents and other business interruptions. 
+
+The unavailability of the pricing service will materially and adversely disrupt the ridesharing network by disabling scheduling of new trips, which may result in an increased number of underutilized aircraft, vertiports, pilots, and staffs. Our customer base may experience churns as a result of unavailable services. Additional costs may be incurred to fix the outage and to resume the network activity.
```

Comment:
This is a fascinating topic from an operations standpoint. We'll need to define our activities in these instances, in some cases manually cancel flights or take a profit loss intentionally to enable refugees to evacuate

Also what happens when the region's currency becomes highly unstable? All of that infrastructure built, theoretically people are relying on our services, etc.

Reply:
This was defined in the high-level CONOPS right? Pricing shouldn't matter much because it doesn't interface clients directly. As soon as svc-cargo/rideshare/charter stops accepting activities, pricing shouldn't get called.

And, for the profit loss case, svc-cargo/rideshare/charter can just skip the pricing call and go directly to confirm flights.

For the unstable currency case, I think pricing cannot do much since everything's denominated in local currencies. And we can just give ppl alternative payment methods like USDC. But the tricky thing is that we would still need to do currency conversion to USD, so we can't really escape the unstable pricing since raw inputs like electricity price are charged in local currencies. 

These are extreme political and economic events, which shouldn't happen often. We can have a crisis-response team that evaluate the situation quickly, and disable the dynamic pricing and switch to manually supplied fixed quotes in USD.